tldw_Server_API/app/core/Ingestion_Media_Processing/Audio/Audio_Files.py

```
@@ -303,12 +306,11 @@ def download_audio_file(url: str, target_temp_dir: str, use_cookies: bool = Fals
         raise
     except requests.exceptions.Timeout:
          logging.error(f"Timeout occurred while downloading audio file: {url}")
-         raise requests.RequestException(f"Download timed out for {url}")
+         raise AudioDownloadError(f"Download timed out for {url}")
```

Ruff (0.14.3)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)